.github/CODEOWNERS

```diff
@@ -273,60 +174,41 @@
-/packages/vscode-extension/src/debug/officeTaskHandler.ts @swatDong @jayzhang @tecton
-/packages/vscode-extension/src/debug/taskTerminal/officeDevTerminal.ts @tecton @swatDong
-/packages/vscode-extension/src/handlers @tecton @jayzhang @HuihuiWu-Microsoft
-/packages/vscode-extension/src/handlers/accounts @kimizhu @swatDong @kuojianlu @a1exwang @qinezh @XiaofuHuang @xiaolang124 @dooriya @HuihuiWu-Microsoft @jayzhang
+/packages/vscode-extension/src/commonlib @HuihuiWu-Microsoft @sherylueen @wh-alice
+/packages/vscode-extension/src/debug @qinzhouxu @llhmm01 @wh-alice
 /packages/vscode-extension/src/handlers/aadManifestHandlers.ts @blackchoey @wenytang-ms @KennethBWSong
+/packages/vscode-extension/src/handlers/accounts @HuihuiWu-Microsoft @sherylueen @wh-alice
 /packages/vscode-extension/src/handlers/collaboratorHandlers.ts @KennethBWSong @SLdragon
-/packages/vscode-extension/src/handlers/copilotChatHandlers.ts @yuqizhou77 @Alive-Fish @tecton
-/packages/vscode-extension/src/handlers/manifestHandlers.ts @nliu-ms @yuqizhou77 @anchenyi
-/packages/vscode-extension/src/releaseBasedFeatureSettings.ts @yuqizhou77 @tecton @Alive-Fish
-/packages/vscode-extension/src/migration @kimizhu @swatDong @kuojianlu @a1exwang @qinezh @XiaofuHuang @xiaolang124 @dooriya
-/packages/vscode-extension/src/pluginDebugger @jayzhang @nliu-ms
-/packages/vscode-extension/src/officeChat @1openwindow
-/packages/vscode-extension/src/qm @jayzhang
-/packages/vscode-extension/src/telemetry @chagong @tecton @HuihuiWu-Microsoft
-/packages/vscode-extension/src/treeview @tecton @HuihuiWu-Microsoft
-/packages/vscode-extension/test @tecton @HuihuiWu-Microsoft @jayzhang
-/packages/vscode-extension/test/localdebug @kimizhu @swatDong @kuojianlu @a1exwang @qinezh @XiaofuHuang @xiaolang124 @dooriya
-/packages/vscode-extension/test/migration @kimizhu @swatDong @kuojianlu @a1exwang @qinezh @XiaofuHuang @xiaolang124 @dooriya
-/packages/vscode-extension/test/pluginDebugger @jayzhang @nliu-ms
-
-/schemas/* @yuqizhou77 @huimiu
+/packages/vscode-extension/src/officeChat @llhmm01 @tecton @wh-alice
+/packages/vscode-extension/src/pluginDebugger @qinzhouxu @llhmm01 @wh-alice
+/packages/vscode-extension/test/localdebug @qinzhouxu @llhmm01 @wh-alice
+/packages/vscode-extension/test/pluginDebugger @qinzhouxu @llhmm01 @wh-alice
 
 /templates/* @hund030 @eriolchan @huimiu
 /templates/**/ai-assistant-bot @kimizhu @swatDong @kuojianlu
 /templates/**/ai-bot @kimizhu @swatDong @kuojianlu
-/templates/**/declarative-agent-with-action-from-existing-api @yuqizhou77 @Alive-Fish @jayzhang @anchenyi
-/templates/**/declarative-agent-with-action-from-scratch @eriolchan @huimiu @Yimin-Jin
-/templates/**/declarative-agent-with-action-from-scratch-bearer @eriolchan @huimiu @Yimin-Jin
-/templates/**/declarative-agent-with-action-from-scratch-oauth @eriolchan @huimiu @Yimin-Jin
-/templates/**/declarative-agent-basic @eriolchan @huimiu @Yimin-Jin
+/templates/**/basic-custom-engine-agent @adashen @frankqianms
```

Comment:
I remember Frank added this template, you might need to sync on the change

Reply:
These lines are moved because I sorted the list alphabetically. I added the basic CEA template and it will be owned by Frank.
The changes to templates folder except sorting are:
```
/templates/**/office-* @llhmm01 @tecton @wh-alice
/templates/**/spfx-* @HuihuiWu-Microsoft @yuqizhou77 @wh-alice
```